community/examples/gke-tpu-v4-2x2x2/gke-tpu-v4-2x2x2-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+terraform_backend_defaults:
+  type: gcs
+  configuration:
+    # The GCS bucket used for storing terraform state
+    bucket:
+
+vars:
+  # Your GCP Project ID
+  project_id:
+
+  # This should be unique across all of your Cluster
+  # Toolkit Deployments.
+  deployment_name: gke-tpu-v4-2x2x2
+
+  # The GCP Region used for this deployment.
+  # example: us-central2
+  region:
+
+  # The GCP Zone used for this deployment.
+  # example: us-central2-b
+  zone:
+
+  # The number of TPU slices to create
+  num_slices: 1
+
+  # The TPU placement topology for pod slice node pool.
+  tpu_topology: 2x2x2
+
+  # The number of nodes to be created in each nodepool
+  # For TPUs, static_node_count should be num of chips based on topology divided by num chips for the machine type
+  # Reference: https://cloud.google.com/tpu/docs/v4
+  static_node_count: 2
```

Comment:
What would happen if user changes this?

Reply:
This should be based on the TPU verison and topology. For TPU v4 with 2x2x2 topology, this HAS TO BE 2. Details can be found here: https://cloud.google.com/tpu/docs/v4#using-accelerator-config